integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -906,6 +906,26 @@ public void testUpdateImageName() {
     assertTrue(verifyRollingRestartOccurred(podsWithTimeStamps, 1, domainNamespace),
         String.format("Rolling restart failed for domain %s in namespace %s", domainUid, domainNamespace));
 
+    // verify the admin server service created
+    checkServiceExists(adminServerPodName, domainNamespace);
```

Comment:
Use the method checkPodReadyAndServiceExists(adminServerPodName, domainUid, domainNamespace)
to combine  checkServiceExists() and checkPodReady()